src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -4128,7 +4128,7 @@ public static void Write<T>(this IWriter<T> writer, T value)
             SymbolInfo info = new SymbolInfo();
             info = model.GetSymbolInfo(call);
 
-            Assert.IsType<SourceOrdinaryMethodSymbol>(info.Symbol.GetSymbol());
```

Comment:
>IsType [](http://example.com/codeflow?start=19&length=6)

I think there is a helper to check for the base type. IsAssignable or something like that. Consider using it instead.

Reply:
done.